shuup/core/models/_products.py

```diff
@@ -132,6 +133,17 @@ def __str__(self):
 
 
 class ProductQuerySet(TranslatableQuerySet):
+
+    def _get_visible_modes(self):
+        # TODO: What's the point of this?
```

Comment:
Remove the todo comment. Point of this is to exclude variation children from being listed in product listings. :)